sdk/go/pulumi/internals/outputs_test.go

```diff
@@ -0,0 +1,42 @@
+package internals
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/stretchr/testify/assert"
+)
+
+func await(out pulumi.Output) (interface{}, bool, bool, []pulumi.Resource, error) {
+	return awaitWithContext(context.Background(), out)
```

Comment:
Not blocking, but ideally this would be testing the public API:

```suggestion
	result, err := UnsafeAwaitOutput(context.Background(), out)

	return result.Value, result.Known, result.Secret, result.Dependencies, err
```

Reply:
This doesn't appear to have changed.